src/evaluation/WorkspaceManager.jl

```diff
@@ -82,7 +82,7 @@ function create_emptyworkspacemodule(pid::Integer)::Symbol
 end
 
 function default_environment_path(::Nothing)
-    get_pl_env("PLUTO_DEFAULT_ENVIRONMENT_PATH")
```

Comment:
@fonsp sorry I didn't bring this up in last PR. I'm wondering what's the reason to change `PLUTO_PROJECT` to `PLUTO_DEFAULT_ENVIRONMENT_PATH`? I think at least at CLI side, this variable is exactly the same as `JULIA_PROJECT`: it can be used to interact with some environment manager tool (e.g `direnv`) to control the environment variable, but when `pluto --project` is used, it gets overwrite by `--project`. I find `PLUTO_PROJECT` is more consistent with Pkg, which is easier to remember, what do you think?

Reply:
I think this name is short enough to be explicit. If you want to make the environment variable name actually matches the field name in the `struct`s, I think renaming the internal field names is probably better.